src/app.ts

```diff
@@ -1,21 +1,75 @@
+import * as dotenv from "dotenv"
+dotenv.config()
+
+import * as admin from "firebase-admin"
+import { SecretManagerServiceClient } from "@google-cloud/secret-manager"
+import { CoreConfig } from "./core/types_core"
 import * as telegram from "./telegram/bot_telegram"
 import * as discord from "./discord/bot_discord"
 import { TelegramConfig } from "./telegram/types_telegram"
 import { DiscordConfig } from "./discord/types_discord"
+import { Core } from "./core/core"
+import { Translator } from "./core/translator"
 
-const telegramRunning = process.argv.includes("--telegram")
-const discordRunning = process.argv.includes("--discord")
+export const config = new CoreConfig()
 
-if (telegramRunning) {
-  const telegramBot = new telegram.EnforcingTelegramBot(new TelegramConfig())
-  console.log("Started Telegram bot.")
-}
+admin.initializeApp()
```

Comment:
Wait how does it get the credentials now?

Reply:
idk but works

Reply:
lol

Reply:
oke brought it back, just in case

Reply:
alr